Source/Core/isBlobUri.js

```diff
@@ -0,0 +1,29 @@
+/*global define*/
+define([
+        './defined'
+    ], function(
+        defined) {
+    'use strict';
+
+    var blobUriRegex = /^blob:/i;
+
+    /**
+     * Determines if the specified uri is a blob uri.
+     *
+     * @exports isBlobUri
+     *
+     * @param {String} uri The uri to test.
+     * @returns {Boolean} true when the uri is a blob uri; otherwise, false.
+     *
+     * @private
+     */
+    function isBlobUri(uri) {
```

Comment:
Also, is it normal for `uri` to be undefined?  Why wouldn't that be a `DeveloperError`?